plugins/pulp_rpm/plugins/importers/yum/sync.py

```diff
@@ -952,20 +961,22 @@ def __init__(self, importer_id, base_url):
         self.importer_id = importer_id
         self.base_url = base_url
 
-    def add(self, unit):
+    def add(self, unit, path):
         """
         Add the specified content unit to the catalog.
 
         :param unit: A unit being added.
         :type unit: pulp_rpm.plugins.db.models.RpmBase
+        :param path: The path component of the download URL.
```

Comment:
I think this is close, but not quite accurate. It's the relative path within the base path of the repo URL, right? Not the entire path component of the URL?